public/app/plugins/datasource/cloudwatch/datasource.js

```diff
@@ -5,18 +5,18 @@ define([
   'app/core/utils/datemath',
   'app/core/utils/kbn',
   'app/features/templating/variable',
-  './annotation_query',
 ],
-function (angular, _, moment, dateMath, kbn, templatingVariable, CloudWatchAnnotationQuery) {
+function (angular, _, moment, dateMath, kbn, templatingVariable) {
   'use strict';
 
   /** @ngInject */
-  function CloudWatchDatasource(instanceSettings, $q, backendSrv, templateSrv) {
+  function CloudWatchDatasource(instanceSettings, $q, backendSrv, templateSrv, timeSrv) {
```

Comment:
CloudWatchDatasource also needs to implement `targetContainsTemplate`. Its used to display an error if query used by alerting contains a template variable. Ex https://github.com/grafana/grafana/blob/master/public/app/plugins/datasource/graphite/datasource.ts#L140

Reply:
Sorry, I didn't notice. Add it!